src/api/projects.ts

```diff
@@ -0,0 +1 @@
+export const GET_PROJECT_BY_ID = (projectId: number): string => `http://localhost:5030/projects/${projectId}`;
```

Comment:
This `http://localhost:5030` could be on .env file but left it as is as I'm not sure if we wanna run this project elsewhere than local